games/rcon.js

```diff
@@ -0,0 +1,172 @@
+/* *
+ * DiscoBot - Gaymers Discord Bot
+ * Copyright (C) 2015 - 2017 DiscoBot Authors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ * */
+
+const appConfig = require('../index').appConfig;
+const https = require('https');
+
+// Minecraft API Spec available at https://github.com/gaymers-discord/server-rcon
+
+exports.mc_whitelist = function(username, action, callback) {
+
+  let commandData = 'whitelist ';
+
+  if (action == 'add') {
```

Comment:
fixed in https://github.com/gaymers-discord/DiscoBot/pull/306/commits/582709737c4c38363ec6099b2f3ddb0a013a76b6